examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/WildcardFragment.kt

```diff
@@ -204,11 +205,11 @@ class WildcardFragment : Fragment() {
 
     val minIntervalEd = dialogView.findViewById<EditText>(R.id.minIntervalEd)
     val maxIntervalEd = dialogView.findViewById<EditText>(R.id.maxIntervalEd)
-    val keepSubscriptionsEd = dialogView.findViewById<EditText>(R.id.keepSubscriptionsEd)
-    val isFabricFilteredEd = dialogView.findViewById<EditText>(R.id.isFabricFilteredEd)
+    val keepSubscriptionsEd = dialogView.findViewById<Spinner>(R.id.keepSubscriptionsEd)
+    val isFabricFilteredEd = dialogView.findViewById<Spinner>(R.id.isFabricFilteredEd)
```

Comment:
"Ed" means EditText. If you want to change to using Spinner, I think it would be better to change the variable name.